### ISSUE TITLE
[content-init] workspace start is slow, waits between `awaiting seccomp fd` and `supervisor: workspace content available`

### ISSUE THREAD

**Issue by kylos101** [gitpod-io/gitpod#13256]
### Bug description

:wave: sometimes workspace starts can take many minutes, and the delay (for some cases) is inbetween [here](https://console.cloud.google.com/logs/query;cursorTimestamp=2022-09-19T04:44:27Z;query=%225740c71b-c6ea-4735-ae15-93daacc7982e%22%0Atimestamp%3D%222022-09-19T04:44:27Z%22%0AinsertId%3D%22w499gwt4enkk0e51%22;summaryFields=:false:32:beginning:false;timeRange=2022-09-19T03:14:00.000Z%2F2022-09-19T06:32:00.000Z?project=workspace-clusters) and [here](https://console.cloud.google.com/logs/query;cursorTimestamp=2022-09-19T05:24:40Z;query=%225740c71b-c6ea-4735-ae15-93daacc7982e%22%0Atimestamp%3D%222022-09-19T05:24:40Z%22%0AinsertId%3D%220yz13esc7g20wuxy%22;summaryFields=:false:32:beginning:false;timeRange=2022-09-19T03:14:00.000Z%2F2022-09-19T06:32:00.000Z?project=workspace-clusters).

Here is a link for the entire set of [logs](https://cloudlogging.app.goo.gl/EzPMdbLEUfiHEZAo8). You'll see that delay the customer experienced (confirmed in webapp logs where we see browser connection [here](https://cloudlogging.app.goo.gl/Zv8m1dv9ZrqYtX8FA), which was approximately ~40 minutes).

**Definition of done**
1. Add logging and tracing that sits in between these two events (i.e. `workspacekit`, `ws-daemon`, maybe `runInitializer`), so, we can understand why we might be stalling for so long. The logging should likely use exponential backoff, so that if its running repeatedly for 40 minutes, it isn't too verbose.
2. Create a follow-up issue whenever we have enough data from the logs.

### Steps to reproduce

Unclear

### Workspace affected

instance id 5740c71b-c6ea-4735-ae15-93daacc7982e

### Expected behavior

The delay should be way less than ~40 minutes.

### Example repository

n/a

### Anything else?

Run this query to view all instances for this workspace. Some have a null minutes to start, others have a really long time, and the bulk and balance are short (under 2 minutes).

[Node performance](https://grafana.gitpod.io/d/xfpJB9FGz2/node-resource-usage-metrics?orgId=1&from=1663559793162&to=1663574500517&var-datasource=VictoriaMetrics&var-origin_prometheus=&var-cluster=us65&var-job=node-exporter&var-nodepool=All&var-hostname=All&var-device=All&var-interval=2m&var-maxmount=&var-show_hostname=workspace-ws-us65-pool-qd3x&var-total=&var-node=workspace-ws-us65-pool-qd3x) at the time of the event seemed okay:

![image](https://user-images.githubusercontent.com/1272076/192043138-a4cb394d-05d1-4982-9adc-f377d5af25e2.png)

![image](https://user-images.githubusercontent.com/1272076/192043434-73594ca4-6861-4b44-ad8a-3cb14ede92bb.png)

![image](https://user-images.githubusercontent.com/1272076/192043485-2b3d5247-d608-4c83-9cd8-5aa7a79e59af.png)

![image](https://user-images.githubusercontent.com/1272076/192043550-e696a8ac-b1c9-47a4-9263-51eb268014f4.png)

<img src="https://front.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_3r3d6)

**Comment by sagor999** [comment]
I think this is the same issue as https://github.com/gitpod-io/gitpod/issues/12345
It is waiting on prebuild\content initializer to do its thing.

**Comment by utam0k** [comment]
Apparently, not only is it slow, but it can lapse for up to an hour and fail to boot. Perhaps a different cause, but the symptoms are similar.
https://cloudlogging.app.goo.gl/QX4BZ6HK4wBueEpv6

We can see https://github.com/gitpod-io/gitpod/blob/0412c5f4619b58a0bbd270adae3613ef82458e62/components/supervisor/pkg/supervisor/supervisor.go#L1364 on this log, 
but before reaching https://github.com/gitpod-io/gitpod/blob/0412c5f4619b58a0bbd270adae3613ef82458e62/components/supervisor/pkg/supervisor/supervisor.go#L1386 , `supervisor` got `SIGTERM` https://github.com/gitpod-io/gitpod/blob/0412c5f4619b58a0bbd270adae3613ef82458e62/components/supervisor/pkg/supervisor/supervisor.go#L391
I'm not sure who send out `SIGTERM` to `supervisor` :thinking:

**Comment by kylos101** [comment]
I am going to remove this from groundwork for now. 

We experienced a similar issue on Friday, and @iQQBot added a change in https://github.com/gitpod-io/gitpod/pull/13828

It should ship in `gen72`. We can consider adding to groundwork in the future if the issue continues.

**Comment by utam0k** [comment]
@kylos101 Should we mark the issue `blocked`?

**Comment by kylos101** [comment]
:wave: @utam0k generally blocked is used because it's actively being worked (in-progress), but we cannot proceed. In this case, I think leaving in inbox is okay (because we'll see recent comments to get context).

**Comment by utam0k** [comment]
@kylos101 Can we close this issue? I think this issue was resolved with https://github.com/gitpod-io/gitpod/pull/14821

**Comment by kylos101** [comment]
@utam0k I defer to you, you are the expert. :smile: Worst case, we can always reopen. :smile_cat:
